### ISSUE TITLE
Update dDTR version providing IDTA Asset Adminstration Shell v3.1 API

### ISSUE THREAD

**Issue by mkanal** [eclipse-tractusx/item-relationship-service#397]
**As** ... ,
**I want** ... ,
**so that** ... .

## Link 
- blocked by https://github.com/eclipse-tractusx/sig-release/issues/468

## Hints / Details

- ...

### Outcome
- ...
### Acceptance Criteria
- [ ] Update dDTR registry to version providing IDTA Asset Adminstration Shell v3.1

## Out of Scope

- ...

**Comment by jzbmw** [comment]
as far as I'm informed AAS 3.1 was descoped for R24.05
